discovery-provider/src/api/v1/users.py

```diff
@@ -146,19 +145,14 @@ def get(self, id):
         return get_single_user(user_id, current_user_id)
 
 
-@full_ns.route("/handle/<string:handle>")
+USER_HANDLE_ROUTE = "/handle/<string:handle>"
```

Comment:
looks like I might need to update my flake8 plugin to unpack variables being passed into @ns.route() :) (today it parses the raw string to find route parameters and check that they're documented)